### DISCUSSION TITLE
Export configuration

### DISCUSSION THREAD

**Discussion by audiamus** [audiamus/BookLibConnect#7] [Show and tell]
From #6:
> Kannst du eventuell eine Import/Export Möglichkeit für die Profile einbauen? Mir geht es insbesondere um die Authentifizierungsdaten. Somit würde eine Registrierung über deine App oder meine App genügen, um beide nutzen zu können.

_In English as it might be of general interest:_

Some kind of export functionality already exists: 

In the "Profiles" dialog you can switch off encryption for the configuration file. It comes with a security warning now, for good reason. You will get plain json text, with all the goodies. I have already used it, and transferred data between your tool, Libation and multiple instances of mine.     
`C:\Users\<User>\AppData\Local\audiamus\BookLibConnect\config`

**Comment by mkb79** [comment]
Thanks for the information. Then I can start here.

**Comment by Maxklos** [comment]
Running [Version 0.13.1](https://github.com/audiamus/BookLibConnect/releases/tag/v0.13.1) : I want to reinstall windows but can't export/save the Profiles. I tried turning off encryption but no config file appears even after modifying one profile. Any Ideas?
